commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/facade/PipelineFacadeImpl.java

```diff
@@ -125,13 +130,29 @@ public Pipeline copyPipeline(Pipeline pipeline) {
      *             container with paging instead
      */
     @Deprecated
-    @PostFilter("hasPermission(filterObject,'view')")
+    @PostFilter("hasPermission(filterObject,'pipeline.read')")
     @Override
     public List<Pipeline> getAllPipelines() {
         return pipelineDao.getAll();
     }
 
     /**
+     * Returns list of all pipelines persisted in the database for given organization.
+     *
+     * @param organizationName of pipeline
+     * @return list of pipelines
+     * @deprecated performance intensive for many pipelines in DB, use lazy
+     * container with paging instead
+     */
+    @Deprecated
+    @PostFilter("hasPermission(filterObject,'pipeline.read')")
+    @Override
+    public List<Pipeline> getAllPipelines(String organizationName) {
+        // TODO: Add filter to pipeline DAO, to return only pipelines with given organization
```

Comment:
This has to be implemented, see also: https://github.com/UnifiedViews/Core/issues/341